packages/core/tests/js/imports/named-import-simple/run.ts

```diff
@@ -0,0 +1,9 @@
+import { imports, type AstTypes } from '@sveltejs/cli-core/js';
```

Comment:
Are the two `named-import-simple` files a leftover? Whats the difference to the non simple use-case?

Reply:
Ohhh, now i do get it. It's the array. We should probably extract into it's own PR, since there is lot's of places we can and should use this within the codebase.

Great idea, but I don't think it has anything to do with this PR